.github/workflows/qc_checks.yaml

```diff
@@ -274,7 +274,6 @@ jobs:
     runs-on: ubuntu-20.04
 
     needs: [ 'pep_style', 'pre-commit' ]
-    if: github.event_name == 'push'
```

Comment:
@SchrodingersGat I guess you don't wanted to remove that for merge?

Reply:
The various backend tests don't take that long any more, I think it is worth checking that the supported databases are tested on PR as well as push

Reply:
Ok